### ISSUE TITLE
Server Crash - Aggregate Exception (Crashes on reboot)

### ISSUE THREAD

**Issue by xdarkxtriviumx** [StrangeLoopGames/EcoIssues#9938]
Reporting bug after some conversation with support in Discord

I have an Eco server that I have been running at 300x300 dimension settings for a couple of weeks now. Everything was running smoothly, but it had been a week or so (I was informed that this length of a wait between restarts was a bad idea) since the last server restart. I went to perform a restart tonight just for the sake of it, and the server started crashing on startup, with a statement of... 

'Total world volume of 3000 x * 238.609294111111 y * 3000 z = 2147483647 is larger than the max supported size of int.MaxValue (2147483647).'

The world is in fact 9km^2 (with settings of 300x300) and has been running fine until now. During troubleshooting a fresh server download was tested with that world.eco file, and even without the one mod it had (trash command mod) the issue still persisted. Attached will be links to both the crash file, as well as the world.eco file for the server. 
Thanks

Link to crash report: https://pastebin.com/Ptpu4uDm
Link to world.eco file: https://www.dropbox.com/s/czub02ybsx1s95m/world.eco?dl=0

**Comment by D3nnis3n** [comment]
Duplicate of https://github.com/StrangeLoopGames/EcoIssues/issues/9858
Bug is fixed in 7.8+ | And yes, this issue can start to happen only after a reboot, i experienced that, too.
Maybe you even got a backup in where it does not yet happen. Either someone build above the height limit, which for your world size is 238 blocks or a bug has catapulted a vehicle over this border, which causes the same.
